recipe/meta.yaml

```
@@ -51,29 +42,29 @@ outputs:
         - nlohmann_json
         - cpp-expected
         - reproc-cpp 14.2.4
-        - spdlog
+        - reproc 14.2.4
+        - spdlog 1.11.0
         - yaml-cpp
         - termcolor-cpp
         - cli11
-        - fmt
-        - libiconv                               # [osx]
-        - winreg                                 # [win]
+        - fmt 9.1.0
+        - winreg             # [win]
```

```suggestion
        - winreg             # [win]
        - zstd 1.5.2
        - bzip2 1.0
```
https://github.com/mamba-org/mamba/blob/2023.02.03/libmamba/CMakeLists.txt#L432